Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,1066 @@
+{
```

Comment:
<p>What does this mean? Seems that mom6 below works well? So we don't need this caveat about mom6 not working?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I deleted this